packages/fhir-router/src/fhirrouter.ts

```diff
@@ -198,3 +198,9 @@ function parseIfMatchHeader(ifMatch: string | undefined): string | undefined {
   const match = /"([^"]+)"/.exec(ifMatch);
   return match ? match[1] : undefined;
 }
+
+export function getFhirRequestParams<T extends Record<string, string | undefined> = Record<string, string | undefined>>(
```

Comment:
It's about type info, technically a key can be missing from the object and if you forget to check the current types will tell you that something is defined. Using this function will make sure all values are checked for `undefined`

Reply:
I'll move this to another PR potentially since it's not necessarily important for this PR.
Just came up while I was working on checking the parameters in the operation